samples/tasks/scenarios/ovn-sandbox/start_sandbox.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 2,
+    "title": "Start sandbox",
+    "subtasks": [{
+        "title": "Start sandbox on farm 0",
+        "workloads": [{
+            "name": "OvnSandbox.start_sandbox",
+            "args": {
+                "sandbox_start_args": {}
```

Comment:
Do we need the amount of sandbox in "sandbox__start_args"?

Could you also provide an example file which creates sandboxes on two farm nodes?